### PR TITLE
Fixes for small issues found at the hackathon

### PR DESCRIPTION
**Pull request by jistr**

### CODE REVIEW

**Comment by jistr** [comment]
Some of this doesn't render right, i'll fix it.

**Comment by jistr** [comment]
Nevermind, it looks good, i was looking at the online rendering instead of my local one :).

**Review by fao89 — APPROVED** [review]

File: docs_user/modules/con_about-machine-configs.adoc [hunk 1]
```diff
@@ -45,5 +45,5 @@ metadata:
 
 Refer to the https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html[OpenShift documentation for additional information on `MachineConfig` and `MachineConfigPools`]
 
-*WARNING:* Applying a `MachineConfig` to an OpenShift node will make the node
-reboot.
+[WARNING]
+Applying a `MachineConfig` to an OpenShift node will make the node reboot.
```
Comment (klgill) [inline docs_user/modules/con_about-machine-configs.adoc:49]:
```suggestion
Applying a `MachineConfig` to an {OpenShift} node makes the node reboot.
```

File: docs_dev/assemblies/development_environment.adoc [hunk 2]
```diff
@@ -227,7 +208,14 @@ https://openstack-k8s-operators.github.io/data-plane-adoption/dev/#_reset_the_en
 
 === Create a workload to adopt
```
Comment (klgill) [inline docs_dev/assemblies/development_environment.adoc:209]:
```suggestion
=== Creating a workload to adopt
```

File: docs_dev/assemblies/development_environment.adoc [hunk 3]
```diff
@@ -227,7 +208,14 @@ https://openstack-k8s-operators.github.io/data-plane-adoption/dev/#_reset_the_en
...
 
-'''
+To be able to run `openstack` commands from the host without actually
+installing the package and copying the configuration file from the VM
+we will create an alias:
```
Comment (klgill) [inline docs_dev/assemblies/development_environment.adoc:213]:
```suggestion
To run `openstack` commands from the host without 
installing the package and copying the configuration file from the virtual machine, create an alias:
```

File: docs_dev/assemblies/development_environment.adoc [hunk 4]
```diff
@@ -330,15 +318,25 @@ Create a Barbican secret
 openstack secret store --name testSecret --payload 'TestPayload'
 ```
 
-== Performing the Data Plane Adoption
+== Performing the adoption procedure
+
+To make our life easier when performing the adoption procedure, we can
```
Comment (klgill) [inline docs_dev/assemblies/development_environment.adoc:323]: I suggest something like: "To simplify the adoption procedure, copy the deployment passwords that you use in the..."

**Pull request merged**